tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+
+- name: Error on distribution
+  fail:
+    msg: "The distribution {{ ansible_distribution }} {{ ansible_distribution_version }} is not supported"
+  when: (ansible_distribution  != "Ubuntu") and ((ansible_distribution_version == "20.04") or (ansible_distribution_version == "19.04"))              
```

Comment:
Shouldn't this be:

`when: not (ansible_distribution  == "Ubuntu" and ((ansible_distribution_version == "20.04") or (ansible_distribution_version == "19.04")) `

although I would drop support for Ubuntu 19.04, since that version is no longer supported by Ubuntu, so we can just write

`when: not (ansible_distribution  == "Ubuntu" and ansible_distribution_version == "20.04")`

?